### PR TITLE
[UR] Bump tag to 534071e52f84bad1dd7fb210a360414507f3b3ae

### PR DESCRIPTION
**Pull request by fabiomestre**
Pulls in the following fixes to the CUDA and HIP adapters:

* [HIP] Correctly set HIP Kernel Buffer Arguments 
* [HIP] Fix host/device synchronization 
* [HIP][CTS] Fix Device CTS failures 
* [HIP] Fix get mem size segfault 
* [HIP] Implement urMemImageGetInfo
* [HIP] Define all UR entry points
* [CUDA] Add support for binary type query
* [CUDA] Update hint functions to only return warnings instead of errors

### CODE REVIEW

File: sycl/plugins/unified_runtime/CMakeLists.txt [hunk 1]
```diff
@@ -53,14 +53,14 @@ endif()
...
+  # Merge: 2f44433c f65473d9
   # Author: Kenneth Benzie (Benie) <k.benzie@codeplay.com>
-  # Date:   Wed Nov 8 13:32:46 2023 +0000
-  #     Merge pull request #1022 from 0x12CC/l0_usm_error_checking_2
-  #     [UR][L0] Propagate OOM errors from `USMAllocationMakeResident`
-  set(UNIFIED_RUNTIME_TAG ec7982bac6cb3a6b9ed610cd6b7cb41fcbc780dc)
+  # Date:   Fri Nov 10 13:25:42 2023 +0000
+  #     Merge pull request #1044 from aarongreig/aaron/clCTSFixMegaBranch
+  #     [OpenCL] Combined CTS fixes
+  set(UNIFIED_RUNTIME_TAG d164792b78cb1f6e683dd5e484af9ac7fb061b4e)
```
Comment (kbenzie) [inline sycl/plugins/unified_runtime/CMakeLists.txt:63]:
```suggestion
  set(UNIFIED_RUNTIME_REPO "https://github.com/oneapi-src/unified-runtime.git")
  # commit 534071e52f84bad1dd7fb210a360414507f3b3ae
  # Merge: 9fc82304 d164792b
  # Author: Kenneth Benzie (Benie) <k.benzie@codeplay.com>
  # Date:   Wed Nov 15 10:51:54 2023 +0000
  #     Merge pull request #1077 from fabiomestre/fabio/combines_fixes_cuda_hip
  #     [CUDA][HIP] Combined CTS Fixes
  set(UNIFIED_RUNTIME_TAG 534071e52f84bad1dd7fb210a360414507f3b3ae)
```

**Review by kbenzie — CHANGES_REQUESTED** [review]
Now that https://github.com/oneapi-src/unified-runtime/pull/1077 is merged, please pull in the latest sycl branch changes and update `UNIFIED_RUNTIME_REPO` and `UNIFIED_RUNTIME_TAG` as suggested. Update the title/description and make this PR ready for review.

File: sycl/plugins/unified_runtime/CMakeLists.txt (hunk 1 above)
Comment (fabiomestre) [reply sycl/plugins/unified_runtime/CMakeLists.txt]: Done :+1:

**Review by kbenzie — APPROVED** [review]

**Comment by kbenzie** [comment]
@intel/llvm-gatekeepers please merge.

**Pull request merged**